_posts/2019-08-08-write-a-new-post.md

```diff
@@ -88,10 +88,20 @@ If you want to add an image to the top of the post contents, specify the url for
 
 ```yaml
 ---
-image: /path/to/image-file
+image:
+  src: /path/to/image-file
 ---
 ```
 
+### Image alt attribute
+
+If you want to add the "alt attribute" to the image at the top of the post contents, specify it by:
+```yaml
```

Comment:
Almost good enough to be merged, but a blank line should be added before the code block ```` ```yaml ````